### ISSUE TITLE
horusec-cli default flag

### ISSUE THREAD

**Issue by fguisso** [ZupIT/horusec#88]
**What happened**:
```
The folder selected is: [/home/my_project_folder]. Proceed? [Y/n]: YY
ERRO[0001] {HORUSEC_CLI} Error when ask if can run prompt question  error="{HORUSEC_CLI} Operation was canceled by user"
Error: {HORUSEC_CLI} Operation was canceled by user
```

**What you expected to happen**:
The command UI can't set the option, just accept a new option or choose deafult method when no options is setted.

**How to reproduce it (as minimally and precisely as possible)**:
just `make install` and ` horusec start` in my project.

**Anything else we need to know?**:
Probably something is wrong with the Command UI pkg.

**Environment**:
- Horusec version (use `horusec version`): 1.4.0
- Operating System: Arch Manjaro

**Comment by wiliansilvazup** [comment]
Thanks for the report @fguisso !
Since what happened the reason was why we already put the letter Y by default in case you added one more Y getting "YY" as an answer.
We are going to open a correction to improve the treatment, until the merge is carried out, please check if you can use the Horusec with just one letter in the answer.

**Comment by wiliansilvazup** [comment]
Fixed on PR #124
